gadget/update.go

```diff
@@ -530,23 +532,27 @@ func EnsureVolumeCompatibility(gadgetVolume *Volume, diskVolume *OnDiskVolume, o
 	}
 
 	// Check if top level properties match
```

Comment:
nitpick (feel free to ignore): to be honest I'd be great to have a more descriptive comment here, especially now the `if` check is harder to follow, I don't feel like `Check if top level properties match` adds any value in this situation

Reply:
I've changed a bit the comment